test/XRP/xrp-client-integration.test.ts

```diff
@@ -43,33 +43,35 @@ describe('XrpClient Integration Tests', function (): void {
   it('Get Transaction Status - Web Shim', async function (): Promise<void> {
     this.timeout(timeoutMs)
 
-    const transactionHash = await xrpWebClient.send(
+    const transactionResult = await xrpWebClient.sendXrp(
       amount,
       recipientAddress,
       wallet,
     )
     const transactionStatus = await xrpWebClient.getPaymentStatus(
-      transactionHash,
+      transactionResult.hash,
     )
```

Comment:
Couldn't you discard this and just use `transactionResult.status` instead of calculating `transactionStatus`?

Reply:
oh you mean like capitalize on the entire point of this refactor in the first place?  not a bad idea lol

Reply:
changed everywhere possible in this file, nice catch mayukha hahaha